app/services/search_orders.rb

```diff
@@ -19,7 +19,7 @@ def fetch_orders
       includes(:payments, :subscription, :shipments, :bill_address, :distributor, :order_cycle).
       ransack(params[:q])
 
-    @search.result(distinct: true)
+    @search.result(distinct: true).order(params[:q][:s])
```

Comment:
This is strange actually. 
ransack cannot handle the sort when passing the params through the init method (`ransack(params[:q])`), as this seems to be done for `email`, ...